### PR TITLE
Added `Color.from_color_space` classmethods

### PR DESCRIPTION
**Pull request by Matiiss**
Fixes https://github.com/pygame-community/pygame-ce/issues/2121

### CODE REVIEW

**Review by dr0id — APPROVED** [review]
LGTM when you undef the macro as suggested by @yunline

**Review by yunline — APPROVED** [review]
LGTM

File: docs/reST/ref/color.rst [hunk 1]
```diff
@@ -191,6 +191,58 @@
...
+   .. classmethod:: from_cmy
+
+      | :sl:`Returns a Color object from a CMY representation`
+      | :sg:`from_cmy(object) -> Color`
+      | :sg:`from_cmy(c, m, y) -> Color`
+
+      Creates a Color object from the given CMY components. Refer to :attr:`Color.cmy`
+      for more information.
+
+      .. versionadded:: 2.3.0
```
Comment (Starbuck5) [inline docs/reST/ref/color.rst:204]: versionadded is now out of date

**Review by Starbuck5 — APPROVED** [review]

**Comment by Starbuck5** [comment]
@Matiiss the commit history is a bit messy at this point, please squash at your discretion and then merge. :)

**Pull request merged**